docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,203 @@
+
+## How to deploy
+
+1. Clone the `nebula-docker-compose` repository to your host with Git.
+
+    * To install Nebula Graph 1.0, clone the `master` branch.
+
+    ```shell
+    $ git clone https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
+    * To install Nebula Graph 2.0-pre, clone the `v2-preview` branch.
+
+    ```shell
+    $ git clone --branch v2-preview https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
+2. Go to the `nebula-docker-compose` directory.
+
+    ```shell
+    $ cd nebula-docker-compose/
+    ```
+
+3. Run the following command to start all the Nebula Graph services.
+
+    ```shell
+    $ docker-compose up -d
+    ```
+
+    The following information indicates the services have started:
+
+    ```shell
+    Creating nebula-docker-compose_metad2_1 ... done
+    Creating nebula-docker-compose_metad1_1 ... done
+    Creating nebula-docker-compose_metad0_1 ... done
+    Creating nebula-docker-compose_storaged2_1 ... done
+    Creating nebula-docker-compose_graphd_1    ... done
+    Creating nebula-docker-compose_storaged0_1 ... done
+    Creating nebula-docker-compose_storaged1_1 ... done
+    ```
+
+    >**NOTE**: For more information of the preceding services, see [Design and Architecture of Nebula Graph](https://docs.nebula-graph.io/manual-EN/1.overview/3.design-and-architecture/1.design-and-architecture/).
+
+4. Use nebula-console to connect to Nebula Graph.
+
+    Nebula-console is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path we set in the following commands and uses it to connect to the graphd service of Nebula Graph. You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/).
+
+   * For Nebula Graph 1.0:
+
+    ```shell
+    $ docker run --rm -ti --network=host vesoft/nebula-console:nightly -u <user> -p <password> --addr=127.0.0.1 --port=3699
+    ```
+
+   * For Nebula Graph 2.0 pre:
+
+    ```shell
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly -u <user> -p <password> --address=graphd --port=3699
+    ```
+
+    >**NOTE**: By default, the authentication is disabled, and the `-u` and `-p` options are unnecessary. To enable authentication, see [Authentication Configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/4.account-management-statements/authentication/#authentication).
+
+    The following information indicates that you have connected to the Nebula Graph services:
+
+    ```shell
+    Welcome to Nebula Graph (Version 5d10861)
+
+    (user@127.0.0.1) [(none)]>
+    ```
+
+## Check the Nebula Graph service status and ports
```

Comment:
I think this could become Step 4 of the Nebula Graph deployment procedure.  

As a user, I would like to check the deployment result. Telling newcomers to check the service status before they connect to nebula-console or studio would be better. Sometimes, it could help them find problems earlier. 

Besides, the status could help newcomers understand why they must connect to Port 3699 to connect to nebula-console or studio.

Reply:
Good point. This part is indeed an optional step. So we could write it in the steps or the next section behind the steps. 
Since there's a Reading guide question at the beginning of this topic, it might be better that readers can jump to this part on-click. Let's keep it this way.